lib/Dialect/Torch/Transforms/DecomposeComplexOps.cpp

```diff
@@ -1895,6 +1896,37 @@ class DecomposeAtenLeakyReluBackwardOp
 };
 } // namespace
 
+namespace {
+class DecomposeAtenLerpScalarOp : public OpRewritePattern<AtenLerpScalarOp> {
+public:
+  using OpRewritePattern::OpRewritePattern;
+  LogicalResult matchAndRewrite(AtenLerpScalarOp op,
+                                PatternRewriter &rewriter) const override {
+    Location loc = op.getLoc();
+    auto resType = op.getType().cast<ValueTensorType>();
```

Comment:
nit: cast to `BaseTensorType`. Types at this stage are not guaranteed to be `ValueTensorType`s.